dot-net-sdk/store/ConfigurationStore.cs

```diff
@@ -79,6 +105,11 @@ public void FetchConfiguration()
         experimentConfigurationResponse.Flags.ToList()
             .ForEach(x => { this.SetExperimentConfiguration(x.Key, x.Value); });
 
+        if (experimentConfigurationResponse.Bandits != null)
+        {
+            this.SetBanditFlags(experimentConfigurationResponse.Bandits);
```

Comment:
Confirming this fully replaces previously loaded bandit flags?

(What we don't want is a bug where removed flags hang around forever because they are never overwritten)

Reply:
👍It's just the one key/object being stored